sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.azure.messaging.eventhubs.checkpointstore.redis;
+
+import com.azure.messaging.eventhubs.CheckpointStore;
+import java.util.List;
+import java.util.Objects;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/*
+Added this file so that an execuatable jar is produced for the build.
+ */
+
+/**
+ * Implementation of CheckpointStore that uses Azure Redis Cache.
+ */
+public class RedisCheckpointStore implements  CheckpointStore{
+    public Flux<PartitionOwnership> claimOwnership(List<PartitionOwnership> requestedPartitionOwnerships){
```

Comment:
All public and protected classes, interfaces, methods, and fields that aren't in an `implementation` package should have javadocs,  and generally it's best to include javadocs even if they're in `implementation`.

Reply:
Thanks! I'll keep it in mind for next time too.